src/app/pages/overlays/basic-bottom-sheet/bottom-sheet/bottom-sheet.html

```diff
@@ -0,0 +1,16 @@
+<mat-nav-list>
```

Comment:
The styling on this seems a little off (font weight, margins) compared to the existing example:
![image](https://user-images.githubusercontent.com/29152776/87301384-7ea85900-c4dd-11ea-979e-d78003fd2c4b.png)
The highlight on hover should extend to the edge of the bottomsheet

Reply:
The font weight should be able to be overwritten with a class, or you could use the subtitle instead of title (unless title is required).